core/src/main/java/io/questdb/ServerMain.java

```diff
@@ -271,17 +277,19 @@ public synchronized void start(boolean addShutdownHook) {
     private void addShutdownHook() {
         Runtime.getRuntime().addShutdownHook(new Thread(() -> {
             try {
+                System.err.println("SIGTERM received");
```

Comment:
why is it twice? it's the same above.

Reply:
ignore me, it's to out vs err

Reply:
one for STDERR and another for SDTOUT

Reply:
yeah, my eyes failed me when I looked at this for the first time.